django_components/templatetags/component_tags.py

```diff
@@ -160,99 +162,177 @@ def __init__(self, slot: "SlotNode", context: Context):
     def default(self) -> str:
         return mark_safe(self._slot.nodelist.render(self._context))
 
+    # (lemontheme): If 'default' is accepted as option keyword for slot tags,
+    #  then I propose 'original' replaces 'default' in the 'slot.super' sense.
+    #  Otherwise, we would have both
+    #  - {% slot ... default %} ...
+    #  and
+    #  - {% fill ... as <alias> %} <alias>.default {%endfill %}.
+    #  I'm afraid this risks being a source of confusion, as users might wrongly assume
+    #  that there is a link between the two separate usages of 'default'.
+    @property
+    def original(self) -> str:
+        return self.default
+
```

Comment:
Not sure I think this is worth changing concepts for. The difference here is "default slot" vs "default content" which are different concepts. Your call, but is this worth having people change their code for? Should we change the docs?

Reply:
I think it's best not overload terms with separate meanings, even if the differences are clear to us. On the other hand, it's probably not worth breaking people's code over.

Which of these options makes most sense to you:

- The _default_ property is maintained but a new _original_ alias is introduced. (Hell, we could even go back to using _super_ =p )
- We use a different name for the slot option. I'm not sure what that would be though.
- We leave things as they are, but add a clarifiying paragraph in the docs. Maybe create a new FAQ section at the end explaining possible sources of confusion.

Reply:
I'm really not worried about this confusion. Default is a very commonly used term and being constrained to only use it in one place in the API feels limiting. It's like saying "parameter" could only be used once. But all sorts of functions take parameters.

I would say "We leave things as they are, and add things to a FAQ _IF_ it's a common confusion from users" is the best option here. But sure, it doesn't hurt to add it the FAQ if you feel strongly about it.

Reply:
Eh, you're probably right. Until others have had the chance to use this new feature, it's too early to say what will or won't be experienced as confusing. Makes sense to hold off on updating the docs until we've received some feedback. I've dropped these lines in a subsequent commit.